File: test-crash-token.js
const { spawn } = require('child_process');
const fs = require('fs');
const config = JSON.parse(fs.readFileSync(process.env.HOME + '/Library/Application Support/cozea/config.json', 'utf8') || '{}');
const profile = Object.values(config.profiles || {})[0];
const token = profile?.preferences?.radonToken;
if (!token) { console.error('NO TOKEN'); process.exit(1); }

const child = spawn('resources/radon/dist/simulator-server-macos', ['ios', '--id', '571DBC5E-01D2-4C67-939C-C620DAC7D085', '-t', token], {
  stdio: ['pipe', 'pipe', 'pipe']
});
child.stdout.on('data', d => console.log('OUT', d.toString()));
child.stderr.on('data', d => console.log('ERR', d.toString()));
child.on('exit', (code, signal) => console.log('EXIT:', code, signal));
setTimeout(() => child.kill(), 5000);

File: test-crash-token.py
import subprocess
import time

token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.e30.t-IDcSemACt8x4iTMCda8Yhe3iZaWbvV5XKSTbuAn0M' # Fake
with open('/Users/admin/Library/Application Support/cozea/settings.json') as f:
    import json
    data = JSON.parse(f.read() || '{}')
    print(data)

p = subprocess.Popen(['resources/radon/dist/simulator-server-macos', 'ios', '--id', '571DBC5E-01D2-4C67-939C-C620DAC7D085', '-t', token],
                     stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)

print("Started PID:", p.pid)
time.sleep(5)
p.terminate()
print("Return code:", p.returncode)
